nornir/plugins/inventory/netbox.py

```diff
@@ -183,10 +243,29 @@ def __init__(
                     host["data"][cf] = value
                 host["data"].pop("custom_fields")
 
+            # Add secrets to host if it has one in Netbox
+            if nb_private_key:
+                # Filter secrets based on netbox secret role slug
+                f_secrets = [s for s in nb_secrets if s.get("role").get("slug") == nb_cred_role_slug]
+                # Add secrets to device if they exist
+                host_secret = [s for s in f_secrets if s.get("device").get("id") == dev.get("id")]
+                if host_secret:
+                    host_secret = host_secret[0]
+                    host["username"] = host_secret.get("name")
+                    host["password"] = host_secret.get("plaintext")
+
+            # management port from netbox config_context if it's available
+            if host.get("data").get("config_context").get("nornir").get("port"):
+                host["port"] = host.get("data").get("config_context").get("nornir").get("port")
+
+            # Attempt to add 'platform' based of value in 'slug'
+            host["platform"] = dev["platform"]["slug"] if dev["platform"] else None
```

Comment:
Why would you set the platform attribute to the slug? The default behavior has always been to set it to the name attribute of the platform in Netbox. I don't see why we should change that.

Reply:
Fixed that. That was a copy/paste from the old class with_slugs. You were already setting platform above. I am removing this.